03_launch_mgmt_cluster.sh

```diff
@@ -185,22 +184,10 @@ function launch_baremetal_operator() {
     update_images
     popd
 
-    if [ "${BMO_RUN_LOCAL}" = true ]; then
-      touch bmo.out.log
-      touch bmo.err.log
-      kustomize build "$kustomize_overlay_path" | kubectl apply -f-
-      kubectl scale deployment metal3-baremetal-operator -n metal3 --replicas=0
-    fi
-
     if [ "${BMO_RUN_LOCAL}" = true ] || [ "${EPHEMERAL_CLUSTER}" = kind ]; then
```

Comment:
No need for the RUN_LOCAL condition here.

Reply:
Removed

Reply:
you need to remove the BMO_RUN_LOCAL condition here, otherwise you will have two Ironic instances when running with Minikube